data_measurements/dataset_statistics.py

```diff
@@ -617,16 +615,9 @@ def prepare_zipf(self):
         self.z.calc_fit()
         self.zipf_fig = zipf.make_zipf_fig(self.z)
 
-def _set_idx_col_names(input_vocab_df):
-    if input_vocab_df.index.name != VOCAB and VOCAB in input_vocab_df.columns:
-        input_vocab_df = input_vocab_df.set_index([VOCAB])
-        input_vocab_df[VOCAB] = input_vocab_df.index
-    return input_vocab_df
-
 def dummy(doc):
```

Comment:
Repeating from Slack convo, I think that was originally from @yjernite to help ensure that there was no additional tokenization (since we pass in something that is already tokenized). Not sure...it's been there since v0.
I'll do a clean-up pass in a bit and see what happens when I remove it....